internal/apiserver/route_get_contract_interface.go

```diff
@@ -28,11 +28,11 @@ import (
 
 var getContractInterface = &oapispec.Route{
 	Name:   "getContractInterface",
-	Path:   "namespaces/{ns}/contracts/interfaces/{interfaceId}",
+	Path:   "namespaces/{ns}/contracts/interfaces/{interfaceID}",
 	Method: http.MethodGet,
 	PathParams: []*oapispec.PathParam{
 		{Name: "ns", ExampleFromConf: config.NamespacesDefault, Description: i18n.MsgTBD},
-		{Name: "interfaceId", Example: "interfaceId", Description: i18n.MsgTBD},
+		{Name: "interfaceID", Description: i18n.MsgTBD},
```

Comment:
I think we've used `Id` consistently in APIs/JSON so far, rather than `ID` - worth a tweak here?